Description: Don't use ant.lib.path (tasks are included
 in global classpath)
Forwarded: no
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2010-02-27
--- a/build/build-properties.xml
+++ b/build/build-properties.xml
@@ -150,18 +150,11 @@
 			location="${aspectj.modules.lib.dir}/ant/lib/ant.jar"/>
 		<taskdef resource="org/aspectj/internal/tools/ant/taskdefs/taskdefs.properties"
               classpath="${aspectj.build.jar}"/>
-		<path id="ant.lib.path">
-			<fileset dir="${aspectj.modules.lib.dir}/ant/lib">
-				<include name="**/*.jar"/>
-			</fileset>
-			</path>
 		<taskdef name="junit"             
-			classname="org.apache.tools.ant.taskdefs.optional.junit.JUnitTask"
-			classpathref="ant.lib.path"/>
+			classname="org.apache.tools.ant.taskdefs.optional.junit.JUnitTask"/>
 	    <available classname="org.apache.xalan.processor.TransformerFactoryImpl" 
 	        property="xalan.available"
 	        value="true">        
-	        <classpath refid="ant.lib.path"/>
 	    </available>
         <condition property="junitreport.available" value="true">
         	<istrue value="${xalan.available}"/>
@@ -173,8 +166,7 @@
     <target name="init-junitreport" depends="init-properties"
         if="xalan.available">
 		<taskdef name="junitreport"             
-			classname="org.apache.tools.ant.taskdefs.optional.junit.XMLResultAggregator"
-			classpathref="ant.lib.path"/>
+			classname="org.apache.tools.ant.taskdefs.optional.junit.XMLResultAggregator" />
     </target>
 
     <target name="init-directories" 
